aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -733,6 +733,14 @@ std::tuple<Tensor, Tensor> _solve_helper_cpu(const Tensor& self, const Tensor& A
 
 // Supports arbitrary batch dimensions for self and A
 std::tuple<Tensor,Tensor> solve(const Tensor& self, const Tensor& A) {
+  TORCH_WARN_ONCE(
+    "torch.solve is deprecated in favor of torch.linalg.solve",
+    "and will be removed in a future PyTorch release.\n",
+    "torch.linalg.solve has its arguments reversed and does not return the LU factorization.\n",
```

Comment:
What should users do if they want the LU factorization?

Reply:
I agree with you, but should we point people to lu and lu_solve here?

Reply:
Would not hurt, I guess :)